vwr/driver.py

```diff
@@ -59,18 +55,18 @@ def _reconnect(self):
         if self.reconnect_trials < 10:
             try:
                 self._connect()
-                self.reconnect_loop.stop()
+                self.get_setpoint()
+                self.reconnect_loop = Timer(self.initialDelay, self._reconnect)
                 self.reconnect_trials = 0
             except:
```

Comment:
I'd specify whatever exception type happens when disconnected. This ensures that this loop doesn't accidentally silence other exception types.